src/Symfony/Bundle/TwigBundle/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ CHANGELOG
 5.0.0
 -----
 
+ * Add event dispatcher to the config, by default escaped to null. related to: https://github.com/twigphp/Twig/pull/3207
```

Comment:
please remove the URL, we don't add these - the message should be self-explanatory

Reply:
Change the message, I hope this description will make it much better ;)